lib/talkable/integration.rb

```diff
@@ -12,8 +12,24 @@ def register_event(params = {})
       register_origin(Talkable::API::Origin::EVENT, params)
     end
 
+    def approve_referral(origin_slug)
```

Comment:
I like the idea of shortcuts but working with referrals isn't a part of integration (in Talkable terms) so I would move them to separate file